package.json

```diff
@@ -14,8 +14,9 @@
 		"treeify": "^1.0.1"
 	},
 	"scripts": {
+		"clean": "lerna clean",
 		"scaffolding": "node ./scripts/helper.js scaffolding",
-		"bootstrap": "lerna bootstrap",
+		"bootstrap": "lerna bootstrap --loglevel verbose",
```

Comment:
I think this might be an oversight?

Reply:
yeap... my bad. I think it was from right at the beginning when I was trying to understand how lerna works